graylog2-web-interface/src/components/event-notifications/event-notification-form/EventNotificationFormContainer.tsx

```diff
@@ -115,21 +117,25 @@ class EventNotificationFormContainer extends React.Component<
     if (action === 'create') {
       promise = EventNotificationsActions.create(notification);
 
-      promise.then(
-        () => {
-          if (!embedded) {
-            history.push(Routes.ALERTS.NOTIFICATIONS.LIST);
-          }
-        },
-        (errorResponse) => {
-          const { body } = errorResponse.additional;
-
-          if (errorResponse.status === 400 && body && body.failed) {
-            this.setState({ validation: body });
-            EventNotificationFormContainer.scrollToFirstError();
-          }
-        },
-      );
+      promise
+        .then(() => {
+          CurrentUserStore.update(currentUser.currentUser.username);
```

Comment:
There should be a refresh method. In general it is recommend to avoid using stores and use hooks instead. In this way if we change how we get the user state we don't have to change it in multiple places just in the hook